server.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+const app = express();
+
+
+app.get('/', (req, res) => {
+    res.send('Home'); 
+});
+
+app.get('/ping', (req, res) => { 
+    res.send('{ pong: 1 }')
```

Comment:
Could you make this an actual json? I think it's a text now, right?

Reply:
yes it's text.
sure I will